system.json

```diff
@@ -48,10 +48,8 @@
   "socket": true,
   "gridDistance": 2,
   "gridUnits": "yrd",
-  "primaryTokenAttribute": "oWind",
-  "secondaryTokenAttribute": "oStrain",
-  "url": "https://github.com/Dulux-Oz/DeadlandsClassic",
-  "manifest": "https://raw.githubusercontent.com/Dulux-Oz/DeadlandsClassic/master/system.json",
-  "download": "https://github.com/Dulux-Oz/DeadlandsClassic/releases/download/0.1/deadlands_classic.zip",
+  "url": "https://github.com/Ampersandrew/deadlands-classic",
+  "manifest": "https://raw.githubusercontent.com/Ampersandrew/deadlands-classic/CombatTracker/system.json",
+  "download": "https://github.com/Ampersandrew/deadlands-classic/releases/download/0.1/deadlands_classic.zip",
```

Comment:
Now that I know what I'm looking at, these values are pointing to your github repo not this repo 🤔

Reply:
I was trying to get the thing to install from github. That was part of me trying to get it to work. It only does that when your module is registered with Foundry. I forgot this was in here. It should be backed out.

Reply:
After I wrote that I realised I should be using a local checkout of the code, not telling it to install from the `system.json` link. So there's no rush, bigger problems to fix than this right now 😅